src/js/components/DataTable/ExpanderCell.js

```diff
@@ -35,11 +45,54 @@ const ExpanderControl = ({ context, expanded, onToggle, pad, ...rest }) => {
   );
 
   if (onToggle) {
+    const expandText =
+      messages?.expand ||
```

Comment:
Do we need this `||` condition or can it simply be `format({ ...})`?

Reply:
Shouldn't need these checks. `format` was designed with the expectation that at least the base messages from grommet has the messages